data/hai/hai reveal 6 end.txt

```diff
@@ -0,0 +1,672 @@
+		"required reputation" 100
+
+	
+event "lofty goals"
+	system "Al Hurr"
+		pos -432 -491
+		government "Pirate"
+		belt 1312
+		trade "Clothing" 201
+		trade "Electronics" 658
+		trade "Equipment" 484
+		trade "Food" 280
+		trade "Heavy Metals" 850
+		trade "Industrial" 621
+		trade "Luxury Goods" 1134
+		trade "Medical" 582
+		trade "Metal" 339
+		trade "Plastic" 353
+		object
+			sprite star/f5
+			period 10
+		object
+			sprite planet/rock15
+			distance 382
+			period 52.6472
+		object
+			sprite planet/rock6
+			distance 686
+			period 77.7827
+		object Aspiration
+			sprite planet/ice8-b
+			distance 980
+			period 98.6622
+		object
+			sprite planet/dust6
+			distance 1299
+			period 119.0528
+		object
+			sprite planet/gas1
+			distance 1706.99
+			period 557.553
+			object
+				sprite planet/dust1
+				distance 216
+				period 11.1964
+	planet Aspiration
+		attributes pirate
+		landscape land/sea17-harro
+		description `The world of Aspiration appears to be a mild, habitable world, but one currently in an advanced ice age now slowly retreating. The equatorial region of the planet is warm and comfortable, a cool temperate at the present time. The native biosphere seems to be mostly grasses and something resembling pine trees. In a few thousand years it might be not dissimilar from Earth before it was paved over, but for now it's a sparse world, with a single major town built on a promontory overlooking a wide river mouth carrying runoff from continental scale glaciers to the sea.`
+		spaceport `The spaceport on Aspiration is a spaceport in the same way that an airport at a regional town is an airport. It has the most minimal facilities to handle a ship landing without disturbance to the nearby town, and that is it. There are no shops or stalls, Aspiration is not on any trade routes and it does not expect visitors. A lone freighter is parked up permanently on the outskirts.`
+		"required reputation" 100
+
+
+mission "Hai Reveal [D05] Cleanup"
+	landing
+	name "Loose Ends"
+	description "Return to the summit and assure Sayari and Xilin that there will be no further problems."
+	source "Aspiration"
+	destination "Mountaintop"
+	clearance
+	to offer
+		has "Hai Reveal [D04] Cleanup: done"
+	on offer
+		conversation
+			`	The world of Aspiration appears to be a mild, habitable world, but one currently in an advanced ice age now slowly retreating. The equatorial region of the planet is warm and comfortable, a cool temperate at the present time. The native biosphere seems to be mostly grasses and something resembling pine trees. In a few thousand years it might be not dissimilar from Earth before it was paved over, but for now it's a sparse world, with a single major town built on a promontory overlooking a wide river mouth carrying runoff from continental scale glaciers to the sea.`
```

Comment:
This happens on landing, so it occurs before you have any chance to read the description.

Reply:
True, but that doesn't change that it's repeated. If other folks are cool with the repetition then I'm cool with resolving, but I'd personally prefer to avoid a such an exact copy-paste. Maybe we can just reword some of it so it's at least not writing something from scratch?